.bumpversion.cfg

```diff
@@ -43,3 +43,15 @@ values =
 [bumpversion:file:model_hub/examples/huggingface/text-classification/glue_config.yaml]
 
 [bumpversion:file:model_hub/examples/huggingface/text-classification/xnli_config.yaml]
+
+[bumpversion:file:model_hub/examples/huggingface/question-answering/squad.yaml]
```

Comment:
Maybe `[bumpversion:glob:model_hub/examples/huggingface/*/*.yaml]`? We won't need to bother adding each file then.

Reply:
thanks for the suggestion.  i changed bumpversion in #2614